modules/variables.tf

```diff
@@ -209,6 +224,18 @@ variable "allowed_regions" {
   description = "List of AWS regions which DCE Principals have access to. These regions will also be targeted for reset in nuke.yml."
 }
 
+variable "orphaned_accounts_alarm_threshold" {
+  type        = string
+  description = "Alarm when number of orphaned accounts is greater than or equal to this threshold."
+  default     = "1"
```

Comment:
Is it possible to tie the alarm to a percent of total accounts as opposed to a number? Wondering this because alerting on 1 orphaned account by default seems a little aggressive - albeit this feature is disabled by default so that's not too big of a deal.

Reply:
We could probably accomplish that by passing `orphaned_accounts_alarm_threshold` into the `account_pool_metrics` lambda as an environment variable, querying the cloudwatch api for this alarm, and then updating it to be the correct number based on the total number of accounts. However, that adds complexity and feels a bit wonky to me.